Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM golang:1.14.1-alpine3.11 as builder
+FROM --platform=$BUILDPLATFORM golang:1.14.1-alpine3.11 as builder
 
 RUN apk add --update --no-cache ca-certificates tzdata git make bash && update-ca-certificates
 
 ADD . /opt
 WORKDIR /opt
+# Run this before `make token-refresher` to be friendy with Docker image layer cache
```

Comment:
```suggestion
# Run this before `make token-refresher` to be friendy with Docker image layer cache.
```

Reply:
How is that more friendly? Just curious (:

Reply:
It's more friendly because at this point the build didn't diverge into the two different architectures yet and the layer with the dependencies downloaded can be used in each different target arch layer, when `make build` is called.

Reply:
👍🏽